ovs-simple/bin/openshift-sdn-simple-setup-node.sh

```diff
@@ -39,7 +39,21 @@ iptables -I INPUT $lineno -p udp -m multiport --dports 4789 -m comment --comment
 
 
 ## docker
-echo "OPTIONS='-b=lbr0 --mtu=1450 --selinux-enabled'" >/etc/sysconfig/docker
+if [[ -z "${DOCKER_OPTIONS}" ]]
```

Comment:
Could we document in the README or print a warning here when someone has set this variable that they better know what they are doing else openshift-sdn won't work correctly?